### PR TITLE
Metainfo Buckets

### PR DESCRIPTION
**Pull request by egonelbre**
What: 

Why:

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: satellite/satellitedb/metainfo_buckets.go [hunk 1]
```diff
@@ -88,11 +100,23 @@ func (buckets *buckets) List(ctx context.Context, projectID uuid.UUID, opts meta
...
+				limit, 0,
+			)
+		} else {
+			dbxBuckets, err = buckets.db.Limited_Bucket_By_ProjectId_And_Name_LessOrEqual_OrderBy_Desc_Name(ctx,
+				dbx.Bucket_ProjectId(projectID[:]),
+				dbx.Bucket_Name([]byte(opts.Cursor)),
+				limit, 0,
+			)
+		}
+		sort.Slice(dbxBuckets, func(i, j int) bool {
```
Comment (egonelbre) [inline satellite/satellitedb/metainfo_buckets.go]: reverse the order instead of using sort

Comment (mniewrzal) [reply satellite/satellitedb/metainfo_buckets.go]: With just reverse order when we have set `"a", "aa", "b", "bb", "c"` and we have cursor `c`, direction `Backward` and limit `1` result will be `a` not `c`

Comment (egonelbre) [reply satellite/satellitedb/metainfo_buckets.go]: I mean just replacing the sort.Slice

File: satellite/satellitedb/metainfo_buckets.go [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+		if err != nil {
+			return metainfo.BucketList{}, err
+		}
+		result.Items[i] = bucket
+	}
+
+	return result, nil
+}
+
+func reverse(buckets []*dbx.Bucket) {
```
Comment (egonelbre) [inline satellite/satellitedb/metainfo_buckets.go]: reverseBuckets

**Comment by egonelbre** [comment]
Completed in #2460

**Pull request closed without merging**